examples/npm_and_rest/src/ccxt.rs

```diff
@@ -12,7 +12,7 @@ extern "C" {
 }
 
 impl CcxtService {
-    pub fn new() -> Self {
+    pub fn default() -> Self {
```

Comment:
I think this was fine as it was, but if `default` is preferred, we should use the `Default` trait

Reply:
@jstarry Not exactly sure why I did this. It should be possible to have both the `new` constructor and implement `Default` as well (fewer breaking changes that way).

Reply:
Forgot that this was an example 😄. I think it's better to use `Default` that `new` here, though.

Reply:
Ok, do you mind removing the `Option` wrapping now that you dropped the `#[derive(Default)]`? I don't think it's needed after that